content/execution_model.tex

```diff
@@ -71,6 +71,16 @@ \subsection{Invoking OpenSHMEM Operations}
 the symmetric address of object $A$ to an \openshmem operation targeting
 \ac{PE} $i$ results in undefined behavior.
 
+Buffers provided to \openshmem routines are \emph{in-use} until the
+corresponding \openshmem operation has completed at the calling \ac{PE}.
+Updates to a buffer that is in-use, including updates performed through locally
+and remotely issued \openshmemem operations, result in undefined behavior.
+Similarly, reads from a buffer that is in-use are allowed only when the buffer
+was provided as a \CTYPE{const} argument to the \openshmem routine that it is
+in-use by.  Otherwise, the behavior is undefined. Exceptions are made for
```

Comment:
Could we change "...that it is in-use by" to "for which it is in-use"?

Reply:
Yes! Ending this sentence with a preposition was really bugging me. 🤓